server/graphql/v1/mutations/collectives.js

```diff
@@ -518,10 +524,12 @@ export async function archiveCollective(_, args, req) {
     membership.destroy();
   }
 
-  if (collective.type === types.EVENT) {
+  if (collective.type === types.EVENT || collective.type === types.PROJECT) {
     return collective.update({ isActive: false, deactivatedAt: Date.now() });
   }
 
+  // TODO: cascade deactivation to EVENTs and PROJECTs?
+
```

Comment:
I think we should cascade the deactivation, the exception should be not deactivating.